src/mono/mono/eventpipe/ep-rt-mono.c

```diff
@@ -3803,22 +3803,47 @@ mono_profiler_jit_done (
 	MonoMethod *method,
 	MonoJitInfo *ji)
 {
-	if (!EventEnabledMonoProfilerJitDone())
+	if (!EventEnabledMonoProfilerJitDone() && !EventPipeEventEnabledMonoProfilerJitDoneVerbose())
```

Comment:
Should the call to `EventPipeEventEnabledMonoProfilerJitDoneVerbose` be a call to `EventEnabledMonoProfilerJitDoneVerbose` to ensure that any other event sinks are accounted for in this check as well?

Reply:
Will fix it.

Reply:
Fixed.